tests/utils.py

```diff
@@ -15,7 +14,7 @@ def mock_model_numline(node):
     else:
         value = 0
         action_0_prob = 0.1
-    return (action_0_prob, 1 - action_0_prob), state 
+    return np.array([action_0_prob, 1 - action_0_prob]), state 
```

Comment:
isn't this supposed to return a value? Why does it return (prob0, prob1), state?

Reply:
Yes this should be value